frontend/src/components/CreateDatabasePrepForm.vue

```diff
@@ -67,6 +67,21 @@
         </span>
       </div>
 
+      <div v-if="showNewTableName" class="w-full">
```

Comment:
I may just choose to use selectedInstance.engine == 'Mongo'

It's mongo specific, and there is no need to pretend it's general. And it's already strange to mix "showNewTableName with "create-db.new-collection-name"